backend/model/Submission.ts

```diff
@@ -0,0 +1,18 @@
+import mongoose, { Schema, Document, Types } from 'mongoose';
+
+export interface ISubmission extends Document {
```

Comment:
```suggestion
export interface ISubmission {
```
You still haven't remove extends here